agent.go

```diff
@@ -1023,13 +1064,18 @@ func (a *Agent) handleInbound(m *stun.Message, local Candidate, remote net.Addr)
 				return
 			}
 
+			tcpType := TCPTypeUnspecified
+			if networkType == NetworkTypeTCP4 && local.NetworkType() == NetworkTypeTCP4 && local.TCPType() == TCPTypePassive {
```

Comment:
How about IPv6?

```go
if networkType.IsTCP() && networkType == local.NetworkType() && local.TCPType() == TCPTypePassive {
```

Reply:
done, simplified to 
`if local.TCPType() == TCPTypePassive {`

becase local and remote network types should be equal anyway, and if tcpType is set it means it is TCP